### DISCUSSION TITLE
Manage Contributors through the UI

### DISCUSSION THREAD

**Discussion by michael-gvdw** [deployKF/deployKF#210] [Questions & Answers]
Hello,
Would it be possible to manage contributors of a namespce through the UI? Is this something that will be implemented? What roles can I define to users when assigning them to a namespace? Is it limited to "view" and "edit"?

**Comment by ReggieCarey** [comment]
I see the GitOps leanings but I have to agree.  Using a Yaml file to manage a user population smacks of password files and shadow files of the last century.  This is a seriously broken "feature". I feel like it should be top priority to get resolved, but I'm not writing the code.  

The deploykf_core.deploykf_profiles_generator section has three sections users, groups, profiles.  Data must be synchronized between the three sections by hand.  Mine is some 889 lines long.

**Comment by thesuperzapper** [comment]
deployKF is centered around the declarative values, allowing users to create profiles outside the values would break this declarative connection. For example, lots of resource are provisioned when you create a profile or add a user to it.

However, I understand the desire to make onboarding easier. I think in most cases, the friction is coming from the lack of ability to read groups from your auth provider. We had a discussion about this in https://github.com/deployKF/deployKF/issues/216#issuecomment-2430439474, so check that out and give your feedback there.
